validate.go

```diff
@@ -192,6 +194,48 @@ func checkProcess(process rspec.Process, rootfs string) {
 	}
 }
 
+func supportedMountTypes() (map[string]bool, error) {
+	supportedTypes := make(map[string]bool)
+	f, err := os.Open("/proc/filesystems")
```

Comment:
On 06/24/2016 12:12 PM, W. Trevor King wrote:
> So I think we want: * A --host-specific flag that skips this test (at least on Linux, where the test is based on a per-kernel /proc/filesystems). * A platform.os check, because we don't want to be comparing mounts against /proc/filesystems if the container targets Windows. * A skip if platform.os is ‘linux’, but we can't open /proc/filesystems (e.g. because we're running the validator on a Windows host). [1]: #49 (comment) <https://github.com/opencontainers/ocitools/pull/49#issuecomment-216585596>
fixed as your suggestions.

-- 
Ma Shimiao
Development Dept.I
Nanjing Fujitsu Nanda Software Tech. Co., Ltd.(FNST)